source/_components/homekit_controller.markdown

```diff
@@ -0,0 +1,22 @@
+---
+layout: page
+title: "Homekit controller support"
+description: "Instructions how to integrate your Homekit devices within Home Assistant."
+date: 2018-03-19 21:04
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: homekit.png
+ha_category: Hub
+ha_release: 0.66
+ha_iot_class: "Local Polling"
+---
+
+[Homekit](https://developer.apple.com/homekit/) controller integration for Home Assistant allows you to connect Homekit accessories to Home Assistant
+
+There is currently support for the following device types within Home Assistant:
+
+- [Light](../light.homekit_controller)
+
+The component will be automatically configured if the [`discovery:`](components/discovery/) component is enabled. For each detected Homekit accessory, a configuration prompt will appear in the web front end. Use this to provide the Homekit PIN.
```

Comment:
Is there a way to disable Homekit discovery? Or ignore devices? If so, that should be described.

Reply:
Done